Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -109,6 +109,11 @@ public class PushDeploymentController : ApiController
                         break;
                     case FetchDeploymentRequestResult.RanSynchronously:
                         response.StatusCode = HttpStatusCode.OK;
+                        response.Content = new StringContent("Succeeded");
+                        break;
+                    case FetchDeploymentRequestResult.RanSynchronouslyFailed:
+                        response.StatusCode = HttpStatusCode.OK;
```

Comment:
Seems like this should be BadRequest or something similar?

Reply:
Along with error text?

Reply:
I changed `FetchDeploymentRequestResult` to include a StatusText and renamed the enum. Don't know if you approve of the pattern, let me know. Changed to BadRequest, agree it's better.